anomalib/models/reversedistillation/lightning_model.py

```diff
@@ -0,0 +1,133 @@
+"""Anomaly Detection via Reverse Distillation from One-Class Embedding.
+
+https://arxiv.org/abs/2201.10703v2
+"""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import logging
+from typing import Dict, List, Tuple, Union
+
+from omegaconf import DictConfig, ListConfig
+from pytorch_lightning.callbacks import EarlyStopping
+from pytorch_lightning.utilities.cli import MODEL_REGISTRY
+from torch import Tensor, optim
+
+from anomalib.models.components import AnomalyModule
+
+from .torch_model import ReverseDistillationModel
+
+logger = logging.getLogger(__name__)
+
+
+@MODEL_REGISTRY
+class Reversedistillation(AnomalyModule):
```

Comment:
I was thinking about that but it might get tricky as we have this line 
![image](https://user-images.githubusercontent.com/17232914/171865278-5886107f-4d2f-4368-af36-09344e7e9517.png)

Reply:
Maybe we could use a dictionary to map the string containing the specified model name to the class